docs/content/languages/python.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Using Python with Fission"
+weight: 10
+---
+
+Fission supports functions written in Python.  Both Python 2.x and
+Python 3.x are supported.  In this usage guide we'll cover how to use
+this environment, write functions, and work with dependencies.
+
+## Before you start
+
+We'll assume you have Fission and Kubernetes setup.  If not, head over
+to the [install guide]().  Verify your Fission setup with:
+
+```
+fission --version
+```
+
+## Add the Python environment to your cluster
+
+Fission language support is enabled by creating an _Environment_.  An
+environment is the language-specific part of Fission.  It has a
+container image in which your function will run.
+
+```
+fission environment create --name python --image fission/python-env --builder fission/python-builder
+```
+
+## Write a simple function in Python
+
+TODO hello world
+
+## HTTP requests and HTTP responses
```

Comment:
Since most people are asking specifically about these terms maybe replace or combine with `The input and output interface`.